### PR TITLE
clarification on functioning of ranking rules

### LINKED ISSUE #1625: Add clarification on the functioning of ranking rules
From time to time, we receive questions about users confused because they are not getting the desired results by removing some of the ranking rules. It could be useful to add a note or something similar reminding that:
👉  ranking rules are there to sort documents and keep results relevant. If you remove the Typo ranking rule, the returned documents will not be sorted by the number of typos, but with or without the Typo ranking rule, documents with typos will always be candidates.

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes #1625

### CODE REVIEW

File: learn/core_concepts/relevancy.md [hunk 1]
```diff
@@ -15,7 +15,9 @@ Each index possesses a list of ranking rules stored as an array in the [settings
 
 Meilisearch uses a [bucket sort](https://en.wikipedia.org/wiki/Bucket_sort) algorithm to rank documents whenever a search query is made. The first ranking rule is applied to all documents, while each subsequent rule is only applied to documents that are considered equal under the previous rule (i.e., as a tiebreaker).
 
-**The order in which ranking rules are applied matters.** The first rule in the array has the most impact, and the last rule has the least. Our default configuration meets most standard needs but [you can change it](/reference/api/settings.md#update-ranking-rules).
+**The order in which ranking rules are applied matters.** The first rule in the array has the most impact, and the last rule has the least. Our default configuration meets most standard needs, but [you can change it](/reference/api/settings.md#update-ranking-rules).
+
+Deleting a rule will mean that Meilisearch no longer sorts results based on that rule. For example, **if you delete the [typo ranking rule](/learn/core_concepts/relevancy.md#_2-typo), documents with typos will still be considered during search**, but they will no longer be sorted by increasing number of typos.
```
Comment (guimachiavelli) [inline learn/core_concepts/relevancy.md:20]:
```suggestion
Deleting a rule means that Meilisearch will no longer sort results based on that rule. For example, **if you delete the [typo ranking rule](/learn/core_concepts/relevancy.md#_2-typo), documents with typos will still be considered during search**, but they will no longer be sorted by increasing number of typos.
```

**Review by guimachiavelli — APPROVED** [review]
Very minor suggestion on my part, otherwise 🙆‍♂️

**Review by guimachiavelli — APPROVED** [review]
🆗

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**